src/arm/96boards.c

```diff
@@ -34,23 +34,31 @@
 #include "common.h"
 #include "arm/96boards.h"
 
-#define DT_BASE "/sys/firmware/devicetree/base"
+#define DT_BASE "/proc/device-tree"
```

Comment:
[Disclaimer: I don't have any of these boards] Are you sure this won't break other boards supported by this file? Is that the case that this path is now `/proc/device-tree` on all those boards and their respective distros/versions?